supplier/tests/test_supplier.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from conftest import Fields, TestFields
+from supplier.models import Supplier
+
+
+class TestSupplierModel:
+    @pytest.mark.django_db()
+    def test_save_supplier(self, supplier0):
+        """Test - checks if supplier is saved in DB.
+
+        Args:
+            supplier0: fixture from module conftest.py.
+
+        Returns:
+            None.
+        """
+        Supplier.save_supplier(supplier0)
+        assert supplier0 in Supplier.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_supplier(self, saved_supplier0):
```

Comment:
This checks that the right input will give us the right output
how about testing that the *wrong* input will fail? 

every `if` or exception handling in the original function\method should suggest you another test-case to check.

Reply:
i have a test for that in PR #51 
```
 @pytest.mark.django_db()
    def test_fail_to_add_supplier(self, supplier0):
        """Test - checks for adding non valid supplier
        Trying to save supplier with unvalid fields.
        Args:
            None.
        Returns:
            None.
        """
        with pytest.raises(ValidationError):
            supplier0.user_name = ''
            Supplier.save_supplier(supplier0)
```

Reply:
if adding a supplier is a functionality introduced in this PR, the test should be in this PR. 
The PR should include tests for all the functionality it introduced, otherwise, if the "add tests" PR fail, we'll need to change things we approved in this PR, and probably should have not.
what I explained in the meeting wasn't "add tests later", but "add extra functionality later", which translates to "add extra functionality tests later" as well.